src/qibolab/platforms/abstract.py

```diff
@@ -109,6 +109,8 @@ def __init__(self, name, runcard):
 
         self.single_qubit_natives = {}
         self.two_qubit_natives = TwoQubitNatives(0)
+        self._update_called_by_multiqubit = False
```

Comment:
I don't like that fact that we need to add this method.
I think that we should be able to find a solution without adding this method.